test/src/components/campaign/CampaignInfoTest.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { fetch } from 'whatwg-fetch';
+import { shallow } from 'enzyme';
+import { Col } from 'react-bootstrap';
+import CampaignInfo from '../../../../src/components/campaign/campaignInfo';
+
+const mockInfo = {
```

Comment:
This could, but some components (a bunch) need huge campaign or score arrays to render. I'll break them down into smaller parts.